packages/ui/src/components/va-stepper/VaStepper.vue

```diff
@@ -225,6 +179,23 @@ export default defineComponent({
       step,
     })
 
+    const save = (targetIndex: number) => {
+      const save = props.steps[modelValue.value].save;
+      let errored = false;
+      if (!validate() && props.linear) {
+        return;
+      }
+      if (save && !errored) {
+        try {
+          save();
```

Comment:
I would better make a validate method which returns true/false instead of throwing error. The same way we have for inputs.

Reply:
Is my use of adding the isValid function acceptable?

Reply:
@m0ksem  can this be resolved?